python/dgl/distributed/dist_graph.py

```diff
@@ -568,18 +545,12 @@ def _init(self):
         self._client.map_shared_data(self._gpb)
 
     def __getstate__(self):
-        return self.graph_name, self._gpb, self._canonical_etypes
+        return self.graph_name, self._gpb
 
     def __setstate__(self, state):
-        self.graph_name, self._gpb_input, self._canonical_etypes = state
+        self.graph_name, self._gpb_input = state
```

Comment:
This is a bad way of passing `_gpb_input` to `self._init()`. Can we do `self._init(gpb_input)` instead?